packages/read-and-create-calendar-events/server/node-express/route.js

```diff
@@ -10,13 +10,15 @@ exports.readEvents = async (req, res, nylasClient) => {
   if (!user) {
     return res.sendStatus(401);
   }
-
+  const { calendarId, startsAfter, endsBefore, limit } = req.query;
   const events = await nylasClient
     .with(user.accessToken)
     .events.list({
       starts_after: Math.floor(new Date().getTime() / 1000),
-      limit: 20,
-      ...(req.query.calendarId && { calendar_id: req.query.calendarId }),
+      limit: limit || 20,
+      ...(calendarId && { calendar_id: calendarId }),
+      ...(startsAfter && { starts_after: startsAfter }),
+      ...(endsBefore && { ends_before: endsBefore }),
```

Comment:
We don't need this conditional logic, let's assume we'll get all the query params for the demo.